chain-test-utils/src/utils/mod.rs

```diff
@@ -0,0 +1,7 @@
+pub mod seed;
+pub use smoke::R;
+
+#[allow(non_snake_case)] // Allow because R is a type name, the function is snake case anyway
+pub fn new_R_from_random_seed() -> R {
```

Comment:
Maybe come up with a more idiomatic name instead? Or just use the two functions combined, which seem hardly much longer to write than this one function name?

Reply:
Well, it felt like I was gonna be instantiating a generator many times, and maybe we would need to change how it is done later so I wanted to have the simplest possible method to get it.
How does `new_random_generator` sounds?

Reply:
Good enough.